src/main/java/dev/joopie/jambot/music/GuildMusicService.java

```diff
@@ -49,10 +55,7 @@ public void destroyGuildMusicService(final Guild guild) {
 
         if (musicPlayer == null) {
             log.warn("Whaaat? How can we destroy something we expected to be there when it's not?!");
-            return;
         }
-
-        musicPlayer.leave();
```

Comment:
De leave methode bevat zaken om queue en de daadwerkelijke audio player op te ruimen, die nog wel opruimen.

Reply:
I will return this method since we fixed it with transactional on the service